components/02-elements/tab/Tab.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <div
+    :class="['tab', modifier && 'tab--' + modifier]"
+  >
+    <template
+      v-for="tab in tabs"
+    >
+      <button
+        :id="tab.tabId"
+        :key="'tab' + tab.tabId"
+        :class="[
+          'tab__title',
+          tab.tabId === activeTab && 'tab__title--active'
```

Comment:
It looks cleaner this way
```suggestion
          { 'tab__title--active': tab.tabId === activeTab }
```